tools/scope_download_classification.py

```diff
@@ -127,6 +149,8 @@ def download_classification(file: str, gloria, group_ids: list, token: str, star
                 probs += [prb_list]
                 period_origins += [origin_list]
                 periods += [period_list]
+                id_origins += [id_origin_list]
+                ztf_ids += [id_list]
 
             # create dataframe from query results
             sources = pd.DataFrame(
```

Comment:
Can we explore using the pandas json_normalize function instead of manually populating the dataframe?

https://pandas.pydata.org/docs/reference/api/pandas.json_normalize.html

Reply:
or https://pandas.pydata.org/docs/reference/api/pandas.read_json.html

Reply:
@guiga004 Thanks for the suggestion, I'm now using json_normalize and it makes the code more succinct. Let me know what you think of this modification.